src/components/Swap/Input.tsx

```diff
@@ -135,11 +133,11 @@ export default function Input() {
       >
         <ThemedText.Body2 color="secondary" userSelect>
           <Row>
-            <USDC isLoading={isRouteLoading}>{usdc ? `$${formatCurrencyAmount(usdc, 6, 'en', 2)}` : ''}</USDC>
+            <USDC isLoading={isRouteLoading}>{usdc ? `${formatCurrencyAmount(usdc, true)}` : ''}</USDC>
             {balance && (
               <Row gap={0.5}>
                 <Balance color={insufficientBalance ? 'error' : 'secondary'}>
-                  <Trans>Balance:</Trans> <span>{formatCurrencyAmount(balance, 4, i18n.locale)}</span>
+                  <Trans>Balance:</Trans> <span>{formatCurrencyAmount(balance)}</span>
```

Comment:
I'm not sure why this is in a `span` - you may be able to remove that bit.

Reply:
done